packages/tools/client-debugger/client-debugger/api-report/client-debugger.api.md

```diff
@@ -29,7 +29,9 @@ export function closeFluidClientDebugger(containerId: string): void;
 
 // @internal
 export interface ConnectionStateChangeLogEntry extends StateChangeLogEntry<ConnectionState> {
+    changeKind: "attached" | "connected" | "contextChanged" | "disconnected" | "disposed" | "closed";
```

Comment:
I would recommend defining an Enum for this, rather than using an in-line union.

Reply:
Also, I believe the "contextChanged" event on `IContainer` is deprecated. Let's double check the events we're subscribing to and not surface data for any that are deprecated.

Reply:
You are right. "contextChanged" event is deprecated. Removing it and changing to Enumeration.